doc/contributor-how-to/how-to-dev-plugin.md

```diff
@@ -0,0 +1,64 @@
+# How to install QGreenland Custom as a developer
+
+QGreenland Custom is a QGIS plugin for downloading a custom set of data, including data which 
+is not part of the QGreenland Core zip package. This how-to guide walks through how to install the plugin as a developer.
+
+First, clone the QGreenland Custom repo from Faunalia:
+
+```
+git clone https://github.com/faunalia/qgreenland-plugin.git
+```
+
+Use the QGIS GUI to locate the QGreenland Custom directory by going to
+Settings -> User Profiles -> Open Active Profile Folder.
+Then, navigate to the /python/plugins directory, PWD and copy the full filpath. 
+
+Next symlink the plugin repo to the plugin directory:
+```
+ln -s $PWD '/filepath'
+```
+## Installing QGreenland Custom
+
+To install QGreenland Custom, navigate to plugins -> Installed -> Check box on QGreenland -> OK
+
+Instead of restarting QGIS after installing QGreenland Custom, 
+install the plugin, Plugin Reloader. You can do this by navigating to 
+Plugin -> Manage and Install Plugins, then searching for Plugin Reloader in the
+search bar. Install the plugin then press OK.
+
+To use the plugin, navigate to Plugins -> Plugin Reloader -> Configure.
+Select qgreenland-plugin in the drop down. Uncheck the 'run the commands
+below before reloading' option. Leave the 'display' option checked, and press OK.
+
+![configure_plugin](/_images/configure_plugin.png)
+
+Navigate to Plugins -> Plugin Reloader -> Reload Plugin: qgreenland-plugin.
```

Comment:
Maybe this section should be under a subheader "How to re-load QGreenland Custom to see changes"?

Reply:
how about changing the main header of this section to 'Installing and reloading QGreenland Custom'?

Reply:
I think the main header of this file is good; maybe it could be shorter? "How to contribute code to QGreenland Custom"?

But I also think this subsection about reloading should have its own subheader. It would be useful to be able to jump straight to these instructions if a user forgets how to reload.